src/cli/commands/create-config.ts

```diff
@@ -0,0 +1,42 @@
+import {CliCommand} from "./interface";
+import {CommanderStatic} from "commander";
+import logger from "../../logger";
+import fs from "fs";
+import {CliError} from "../error";
+import {writeTomlConfig} from "../../util/toml";
+
+interface ICreateConfigOptions {
+  outputFile: string;
+}
+
+export class CreateConfigCommand implements CliCommand {
+  public register(commander: CommanderStatic): void {
+    commander
+      .command("create-config")
+      .description("Create default config file")
+      .option("-o, --outputFile [output_file]", "Path to output file destination")
```

Comment:
Should add default output file, perhaps `.option("-o, --outputFile [output_file]", "Path to output file destination", "lodestar-config.toml")`

Reply:
awesome, yeah I'll do that now. Do you want it to default to a file if the -o / --outputFile flag is not set at all? IE: `./bin/lodestar create-config` would just create using a default file?